core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1994,11 +1994,29 @@ public class SqlStdOperatorTable extends ReflectiveSqlOperatorTable {
   public static final SqlFunction CURRENT_DATE =
       new SqlCurrentDateFunction();
 
+  /** The <code>TIMEADD</code> function. */
+  public static final SqlFunction DATEADD =
```

Comment:
Instead of updating `SqlStdOperatorTable.java` you should be overriding in our own `DatetimeOperatorTable.java` file